connector/src/main/java/com/google/cloud/spark/bigquery/v2/BigQueryEstimatedTableStatistics.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright 2020 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.spark.bigquery.v2;
+
+import com.google.cloud.bigquery.Job;
+import com.google.cloud.bigquery.JobStatistics;
+import com.google.cloud.bigquery.TableDefinition;
+import com.google.cloud.bigquery.TableInfo;
+import com.google.cloud.bigquery.connector.common.BigQueryClient;
+import com.google.cloud.bigquery.connector.common.TableStatistics;
+import com.google.common.collect.ImmutableList;
+import org.apache.spark.sql.sources.v2.reader.Statistics;
+
+import java.util.Optional;
+import java.util.OptionalLong;
+
+class BigQueryEstimatedTableStatistics implements Statistics {
+
+  private static BigQueryEstimatedTableStatistics UNKNOWN_STATISTICS =
```

Comment:
This seems unused with the new logic to compute stats for views.

Reply:
It will be once we will add the guards...